float64[] tau_linkside_filt
float64[] q_dot_filt
float64[] tau_total
float64[] iq_meas
float64[] iq_meas_filt
float64[] tau_friction_meas
float64[] alpha_f0
float64[] alpha_f1
float64[] K_d0_cal
float64[] K_d1_cal
float64 sol_millis

string[] iq_jnt_names
